### DISCUSSION TITLE
Box64 on Archlinux in Termux

### DISCUSSION THREAD

**Discussion by DocMAX** [ptitSeb/box64#1353] [General]
So i installed Arch Linux with proot-distro and installed box64-git from the AUR repo. I know 32bit isn't possible so i use wow64 version of Wine 9.4
Now why i am facing this:
```
Using native(wrapped) libXdmcp.so.6
Using native(wrapped) libXinerama.so.1
Using native(wrapped) libXxf86vm.so.1
Using native(wrapped) libXrender.so.1
Using native(wrapped) libXrandr.so.2
Using native(wrapped) libXcomposite.so.1
Using native(wrapped) libXi.so.6
Using native(wrapped) libXcursor.so.1
Using native(wrapped) libvulkan.so.1
Using native(wrapped) libGL.so.1
00e8:err:d3d:wined3d_caps_gl_ctx_create Failed to find a suitable pixel format.
00e8:err:d3d:wined3d_adapter_gl_init Failed to get a GL context for adapter 003D5060.
```
My guess is missing x86 libraries in /usr/lib/x86_64-linux-gnu/ like a x86 version of libgl.1.x.
Can someone help?

**Comment by ptitSeb** [comment]
Box64 doesn't use x86_64 libGL, but the arm64 version of it directly, and it seems to have loaded correctly.
Aren't you missing some X11 something? Or maybe you are missing a proprer OpenGL driver for your phone, like Zink?

**Comment by ptitSeb** [comment]
> Box64 doesn't use x86_64 libGL, but the arm64 version of it directly, and it seems to have loaded correctly. Aren't you missing some X11 something? Or maybe you are missing a proprer OpenGL driver for your phone, like Zink?

tl:dr: your OpenGL is not working.

**Comment by DocMAX** [comment]
In the exact same prompt i can run glmark2 with virgl. So why does box64 have a problem??
![grafik](https://github.com/ptitSeb/box64/assets/5351323/1d5246e6-37f6-4fa7-bfe9-6f218b8b4f2b)

**Reply by ptitSeb** [reply]
And yet, wine doesn't find a glx config. I don't think that's box64. Not sure what kind of game you want to run, but only dx9 max game will work with mesa 3.2. Try to run with `WINEDEBUG=+d3d` to get more detail on the wined3d process, maybe you'll see something there...

**Comment by DocMAX** [comment]
I my fault, i did not use the latest git version of box64. Now runs with wine 9.4 wow64. but "geexlab demos" are empty, don't know why...
mesa opengl is 4.3 via mesa_gl_version_override=4.3.
edit: fix was simple. need to run program exe from the program dir. everything works!
